contents/blog/practical-alerts.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: '6 practical uses for PostHog alerts'
+date: 2024-12-16
+author:
+ - bijan-boustani
+rootpage: /blog
+category: General
+tags:
+  - Alerts
+  - Product updates
+---
+
+To keep life interesting, it's important to live in a state of perpetual ignorance. When things are going well, there's no need to find out why. When things go south, let anxiety run its course.
```

Comment:
I wrote this as a fun, funny introduction. But if it doesn't land then it's no big deal to me if we want to rip it out and start with the "PostHog's new alert features..." line below.